lib/postprocessing/image.js

```diff
@@ -87,23 +85,49 @@ function hasExtraInstructions(instructions) {
 }
 
 async function getImageMetadata(file) {
-    const start = new Date();
     try {
-        // [1x1+0+0] is a trick to avoid reading all pixels into memory, improving time for large images
-        const result = await gm(`${file}[1x1+0+0]`).write("json:-");
-        const json = JSON.parse(result.toString());
+        // suffix attributes with `#` to disable human-readable parsing if exiftool not used with `-n` flag
+        const attributes = [
+            "-Orientation",
+            "-FileType",
+            "-ImageHeight",
+            "-ImageWidth",
+            "-JPEGQualityEstimate"
+        ];
+        const attributesToExtract = attributes.join(" ");
+        const command = `exiftool -n -json ${attributesToExtract} ${file}`;
+        console.log(`Metadata (exiftool) command is: ${command}`);
+
+        const { stdout, stderr } = await exec(command);
```

Comment:
Should I log timing? I don't think it's necessary for exiftool, but open for debate.